src/base/trafficGraph.js

```diff
@@ -756,8 +761,10 @@ class TrafficGraph extends EventEmitter {
 
     _.each(this.nodes, (n) => { n.filtered = false; });
     _.each(this.connections, (c) => { c.filtered = false; });
-    this._updateConnectionFilters(filters);
-    this._updateNodeFilters(filters);
+    if (Object.keys(this.nodes).length > 1) {
```

Comment:
Yeah, it would change `if (node.connected)` to `if (node.connected || forceNodesVisible)`. I don't recall why I made `totalNodes` increment only if it's connected, but it definitely is cleaner if you remove that check; then you don't need to add `forceNodesVisible`